package/MDAnalysis/analysis/density.py

```diff
@@ -306,7 +306,7 @@ def make_density(self):
             warnings.warn(msg)
             return
 
-        dedges = map(np.diff, self.edges)
+        dedges = list(map(np.diff, self.edges))
```

Comment:
On 29/01/16 22:49, kain88-de wrote:
>
> In package/MDAnalysis/analysis/density.py 
> <https://github.com/MDAnalysis/mdanalysis/pull/683#discussion_r51320581>:
>
> > @@ -306,7 +306,7 @@ def make_density(self):
> >              warnings.warn(msg)
> >              return
> >
> > -        dedges = map(np.diff, self.edges)
> > +        dedges = list(map(np.diff, self.edges))
>
> This is a complicated way for a list comprehension |[np.diff(e) for e 
> in self.edges]|
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/MDAnalysis/mdanalysis/pull/683/files#r51320581>.
>
I do not have the reflex yet.